make/test.mk

```diff
@@ -54,8 +54,28 @@ test-ci: setup-integration-tests | $(NEEDS_GOTESTSUM) $(NEEDS_ETCD) $(NEEDS_KUBE
 ## or an apiserver.
```

Comment:
@SgtCoDFish the `test` and `test-ci` targets have not yet been made to work with the new folder structure.
eg. https://storage.googleapis.com/jetstack-logs/pr-logs/pull/cert-manager_cert-manager/5880/pull-cert-manager-master-make-test/1640813903300530176/build-log.txt contains the error `go: warning: "./cmd/..." matched no packages`

Reply:
Ahh great catch, I think I misremembered how tests were run. I'll fix :hammer: